aether-entity-extraction-module/extractor/tests/test_manager.py

```diff
@@ -149,14 +174,23 @@ def test_entity_extraction(self):
         self.assertEqual(len(manager.PROCESSED_SUBMISSIONS), 1)
         self.assertEqual(len(manager.realm_entities[TENANT]), 3)
 
+        manager = ExtractionManager(self.redis)
+        manager.entity_extraction(self.wrong_task)
+        self.assertEqual(len(manager.PROCESSED_SUBMISSIONS), 1)
+        self.assertNotIn(TENANT, manager.realm_entities)
+        with self.assertRaises(Exception):
+            manager.entity_extraction(self.wrong_mapping_task)
+            self.assertEqual(len(manager.PROCESSED_SUBMISSIONS), 2)
+        self.assertEqual(len(manager.PROCESSED_SUBMISSIONS), 1)
+
     @mock.patch(
         'extractor.utils.kernel_data_request'
```

Comment:
It's `'extractor.manager.kernel_data_request'`, please add the following in the test to check that you are using that mocked method. You don't need to include the response

```python
    @mock.patch('extractor.manager.kernel_data_request', side_effect=Exception)
    def test_push_to_kernel__error(self, mock_kernel_data_request):
        manager = ExtractionManager(self.redis)
        mock_kernel_data_request.assert_has_calls([
            mock.call(
                url='entities/',
                method='post',
                data=mock.ANY,
                realm=TENANT,
            ),
            mock.call(
                url='submissions/bulk_update_extracted/',
                method='patch',
                data=mock.ANY,
            ),
        ])
```